### PR TITLE
update jester to 0.4.1

### PR DESCRIPTION
**Pull request by waghanza**
Hi @dom96,

`jester` was de-activated here.

This `PR` fix also #331

I also use `0.4.1` inspired by https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/Nim/jester

Regards,

### CODE REVIEW

**Comment by dom96** [comment]
Cool. Is it working for you now?

File: nim/jester/Dockerfile [hunk 1]
```diff
@@ -1,17 +1,10 @@
-FROM gcc:latest
-
-RUN apt update && apt install -y libgc-dev
-
-ENV CHOOSENIM_NO_ANALYTICS 1
-ENV CHOOSENIM_CHOOSE_VERSION #0c683d28bbd5
-RUN curl https://nim-lang.org/choosenim/init.sh -sSf | sh -s -- -y
-ENV PATH $PATH:/root/.nimble/bin
+FROM nimlang/nim
```
Comment (dom96) [inline nim/jester/Dockerfile]: You should pin to Nim 0.19.0 in case something breaks in the future.

File: FRAMEWORKS.yml [hunk 2]
```diff
@@ -99,9 +99,9 @@ crystal:
     website: spider-gazelle.net
     version: "1.1"
 nim:
-  # jester:
-  #   github: dom96/jester
-  #   version: "0.4"
+  jester:
+    github: dom96/jester
+    version: "0.4"
```
Comment (dom96) [inline FRAMEWORKS.yml]: *0.4.1

Comment (waghanza) [reply FRAMEWORKS.yml]: sure, but in fact this is use on `README` (to display version of each framework).
To clarify/simplify maintainance, I only display them on X.Y format

**Comment by waghanza** [comment]
@dom96 we plan to show only some frameworks by default (`native`, `stable`, with a certains features ...)

this mean `jester` (and `httpbeast`) will **not** be show by default (use `--experimental` to show results)

what do you think bout that ?

we are talking about that here https://github.com/orgs/the-benchmarker/teams/web-frameworks/discussions/2

**Comment by dom96** [comment]
I can understand httpbeast, but why would jester be excluded?

**Comment by waghanza** [comment]
@dom96 because of `version`, <1. and it's **not** an exclusion

**Comment by waghanza** [comment]
@dom96 thinking about it, you're right.
Not showing some framework because of a version contraint will disable many here.

We have plently of frameworks un <1, and their are used in production.

So I've changed my mind, OK to keep `jester` and OK for `httpbeast`

**Comment by dom96** [comment]
Awesome :)

**Comment by waghanza** [comment]
@dom96 I've update the results https://github.com/the-benchmarker/web-frameworks#full-table with `jester`

**Pull request merged**